### ISSUE TITLE
Cannot figure out how to add a word to the dictionary from within Caret

### ISSUE THREAD

**Issue by BruceEckel** [careteditor/caret#652]
I'm running 3.4.6 on Windows 10 and I cannot figure out how to add a word to the dictionary from within Caret. From the Github issues, it seems to be possible but right-clicking on the underlined word doesn't provide the option to add the word, and I can't find anything in the menus or keyboard shortcuts. And there doesn't seem to be a help system.

I tried the beta of 4.0 first but it didn't seem to have a way to widen the text window.

**Comment by borekb** [comment]
Ctrl+Space.

**Comment by astoilkov** [comment]
Hi Bruce,

I have tried to right click on a mistaken word in the official Caret release on Windows 10 but wasn't able to replicate the problem. Can you take a look at the image below that shows the behavior on our test machine? Do you experience something else?

![image](https://user-images.githubusercontent.com/884810/36213481-84423e32-11af-11e8-8e54-3f7a3032833b.png)

Thanks in advance.

**Comment by astoilkov** [comment]
Additionally, you could try our custom context menu as Borek mentioned which can be opened using `Ctrl+Space` or `Alt+Space`.

**Comment by borekb** [comment]
@astoilkov I tried about 10 right-clicks now, some of them were ok, some were just this:

![image](https://user-images.githubusercontent.com/101152/36214591-270df22e-11aa-11e8-8d9c-30b13332b939.png)

Ctrl+Space always works.

**Comment by BruceEckel** [comment]
I see the issue. Sometimes right clicking or control-spacing will highlight
the whole word, in which case you get this for right-clicking:

[image: Inline image 1]

And this for control-space:

[image: Inline image 2]

However, if the word *doesn't* get selected and you right click, you get
the desired:

[image: Inline image 3]

And for control-space:

[image: Inline image 4]

The problem is that for some reason it will often highlight the whole word,
and you don't get the desired result.



-- Bruce Eckel
MindViewLLC.com
blog.BruceEckel.com <http://blog.bruceeckel.com/>
SoftwareArchitectureForum.com <http://www.SoftwareArchitectureForum.com>
WinterTechForum.com <http://www.WinterTechForum.com>
OnJava8.com <http://www.OnJava8.com>
www.AtomicScala.com
Reinventing-Business.com <http://www.Reinventing-Business.com>
TrustOrganizations.com <http://www.ScalaSummit.com>

**Comment by astoilkov** [comment]
Yes. Indeed. I can experience the same problem too. We will fix this behavior in the next beta release.

Thanks for reporting this!
